lib/create-file.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var File = require('vinyl'),
+	fs = require('fs'),
+  fsp = require('fs-promise');
+
+/**
+ * Creates a vinyl file
+ * @function createFile
+ *
+ * @param {STRING}  filePath  relative path to file
+ * @param {STRING}  cwd  current working directory
+ * @param {STRING}  base  directory where we start looking for the file
+ * @param {STRING}  type  stream|null
+ */
+exports.createFile = function createFile (filePath, cwd, base, type) {
```

Comment:
These function names are required. That's how the system exports this file's function. The pointer is in utility.js and removing the exports part causes an error.